docs/cvl/statements.md

```diff
@@ -345,6 +346,30 @@ assert lastReverted, "Expected revert when value exceeds limit";
 
 In this example, the `@withrevert` modifier is applied to the `Deposit` function call, which is expected to revert if the `value` exceeds the specified `limit`. The `assert` statement checks whether `lastReverted` is true, ensuring that the contract execution does revert as anticipated when the condition is violated. The error message in the `assert` provides additional context about the expectation.
 
+Since version TODO, this applies not only for Solidity function calls. CVL functions can now revert as well, and the `revert` statement is available in CVL. CVL functions also set the `lastReverted` variable, just like Solidity functions and reverts of calls without `@withrevert` are propagated up to their callers. Only if no call in the call stack had a `@withrevert` annotation do we assume no revert happened. A call with `@withrevert` stops propagation of reverts and will not make the calling function revert, but instead allows reading the `lastReverted` variable immediately after it to check whether the call reverted.
```

Comment:
```suggestion
Since version 8, this applies not only for Solidity function calls. CVL functions can now revert as well, and the `revert` statement is available in CVL. CVL functions also set the `lastReverted` variable, just like Solidity functions and reverts of calls without `@withrevert` are propagated up to their callers. Only if no call in the call stack had a `@withrevert` annotation do we assume no revert happened. A call with `@withrevert` stops propagation of reverts and will not make the calling function revert, but instead allows reading the `lastReverted` variable immediately after it to check whether the call reverted.
```